Porting/release_managers_guide.pod

```diff
@@ -397,6 +397,35 @@ colon-delimited versions to use for comparison.  For example:
 
 L<http://analysis.cpantesters.org/beforemaintrelease?pair=5.20.2:5.22.0%20RC1>
 
+=head3 check pod errors
+
+F<t/porting/podcheck.t> is a porting test that will fail if it finds new
+problems in pods.  However, it can be taught to ignore problems, and
+sometimes people do so for problems that really should be fixed before
+release.  To see what it is ignoring, run
+
+ perl t/porting/podcheck.t --counts
+
+Any problems listed as pedantic aren't worth your time investigating.
+These have a C<?> at the beginning of the text, or are for the too-long
+verbatim lines.
+
+But other warnings could be.  In particular, a broken link can well
+mean that someone clicking on the pod in a web page will get a 404.
+
+To find out more about any real problems, capture the output from
+
+ perl t/porting/podcheck.t --show-all
+
+and grep for those problems.  (It can take a minute or so to run.)
+
+If you decide any should be fixed, after that happens, run
+
+ perl t/porting/podcheck.t
```

Comment:
As @jkeenan points out, this doesn't work.

Now assuming I do:
```
$ ./perl t/porting/podcheck.t --counts
# 3       ? Should you be using F<...> or maybe L<...> instead of
# 6       ? Should you be using L<...> instead of
# 44      Verbatim line length including indents exceeds 78 by
# -----
# 53      known potential issues
# 
# Files that have all messages of at least one type suppressed:
# ext/pod-html/corpus/perlvar-copy.pod, lib/config.pod, pod/perldebguts.pod, pod/perldtrace.pod, pod/perlperf.pod, pod/perlsolaris.pod, porting/epigraphs.pod
```
instead, how much of that is relevant to the current release?

Are we expecting a release manager to inspect each change each time?

Reply:
Squashing the commits makes clear that the new text says don't bother with any warning which begins with a ? or the Verbatim warnings.  So the example you ran would have nothing else for the release manager to do.  This step under those circumstances would add less than a minute to the release process.

If there are other issues, it would be up to the release manager's judgment as to if they should hold up a release or not.  My guess is that only a few possible things might be deemed to require fixing before proceeding with a development release, but more with a stable release

Reply:
Do you have an example of a problem you think should not have been ignored?  When such a problem perhaps could have been ignored?

I can see deciding this to involve a level of judgement that might be outside of what we expect from a release manager.  Release managers have a fair amount of work to do and while it requires some technical perl knowledge, it's fairly mechanical.

Adding the requirement of perhaps a core developer's level of judgement for this might be too much.

Reply:
The example is included in the text.  A broken link was shipped in a devel release.  The podcheck db had been regenerated to ignore the problem.  Text could be added that if you don't understand the problem, get help from someone who does.  If no such person is immediately available, ignore the issue

Reply:
I was looking for a specific case, which I assume a0d74cd693e9 (break) / 2cc10db0945c (fix) is.

For this particular case I wonder if it would be better to make it non-ignorable and expect it to be added to the links list.

Reply:
In thinking about it more, these errors are just about pod, and I think all our release managers would be familiar with pod.  I don't know all the possible errors that can get generated, but my suspicion is that all the ones not listed as in the text here are likely to need to be fixed.

I'm reluctant to make things non-ignorable, as circumstances could arise that I haven't foreseen, such as it being wrong about the link being broken, and it's best to be humble about that possibility.

@tonycoz  did you really mean to approve this?

Reply:
> @tonycoz did you really mean to approve this?

Yes